src/CPUSolver.cpp

```diff
@@ -1678,22 +1679,17 @@ double CPUSolver::computeResidual(residualType res_type) {
 void CPUSolver::computeKeff() {
 
   double* FSR_rates = _regionwise_scratch;
-  double rates[3];
+  double fission[0];
 
-  /* Loop over all FSRs and compute the volume-integrated total rates */
-  for (int type=0; type < 2; type++) {
+  /* Compute the old nu-fission rates in each FSR */
 #pragma omp parallel for schedule(guided)
-    for (long r=0; r < _num_FSRs; r++) {
+    for (int r=0; r < _num_FSRs; r++) {
```

Comment:
I think it should be long rather than int as _num_fsrs would be huge for real problems.